src/web/app/components/course-edit-form/course-edit-form.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { AjaxLoadingComponent } from '../ajax-loading/ajax-loading.component';
+
+import { CourseEditFormComponent } from './course-edit-form.component';
+
+describe('CourseEditFormComponent', () => {
+  let component: CourseEditFormComponent;
+  let fixture: ComponentFixture<CourseEditFormComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [
+        CourseEditFormComponent,
+        AjaxLoadingComponent,
+      ],
+      imports: [
+        HttpClientTestingModule,
+        FormsModule,
+      ],
+    })
+      .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CourseEditFormComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
Do add some frontend tests (snapshot/unit) for this new `course-edit-form` component, especially since the now-deleted `add-course-form` component had frontend tests as well.

Reply:
Will update the tests.